pkg/mls/store/queries.sql

```diff
@@ -0,0 +1,78 @@
+-- name: GetAllInboxLogs :many
+SELECT * FROM inbox_log
+WHERE inbox_id = $1
+ORDER BY sequence_id ASC FOR UPDATE;
+
+-- name: GetInboxLogFiltered :many
+SELECT a.* FROM inbox_log AS a
```

Comment:
This is probably the fanciest query we have.

We need to take in an array of (`inbox_id`, `sequence_id`) pairs and get the results of each inbox that are greater than the `sequence_id`. We are currently doing this in one query per inbox, but this change makes it all happen in one query by joining against a virtual table.